deploy/olm-catalog/nexus-operator/0.3.0/nexus-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -62,6 +62,9 @@ spec:
         name: A Kubernetes Service
         version: v1
       specDescriptors:
+      - description: Automatic updates configuration
```

Comment:
Could you please also change the capability level to 2? :sunglasses:

Reply:
Hell yeah :sunglasses: